zellij-server/src/pty.rs

```diff
@@ -731,11 +739,13 @@ impl Pty {
                     self.bus
                         .os_input
                         .as_mut()
-                        .unwrap()
+                        .with_context(err_context)
+                        .map_err(|_e|SpawnTerminalError::GenericSpawnError("os input is none"))?
```

Comment:
These two lines should be swapped, i.e. first we convert the error type (with `map_err`), and then we attach the context to that instead.

Reply:
It report error after i swapped it, so i changed it to `ok_or_else`.